src/lib/csv.ts

```diff
@@ -136,6 +138,8 @@ export default async function csv({
       if (!noHeader) {
         const headerNames = order.map((id) => map[id])
         if (headerNames.length > 0) {
+          if(title!=='')
```

Comment:
Strange, this passed lint :/

Reply:
Fixed in #388

Reply:
Oh, can you explain why it shouldn't have passed?

Reply:
There is setup lint and prettier config for consistent formatting, but there was an issue that it was not configured correctly and lint only disabled formatting rules, but didn't used rules from prettier